x/bank/types/send_authorization.go

```diff
@@ -45,5 +74,25 @@ func (a SendAuthorization) ValidateBasic() error {
 	if !a.SpendLimit.IsAllPositive() {
 		return sdkerrors.ErrInvalidCoins.Wrapf("spend limit must be positive")
 	}
+
+	found := make(map[string]bool, 0)
+	for i := 0; i < len(a.AllowList); i++ {
+		if found[a.AllowList[i]] {
+			return ErrDuplicateEntry
```

Comment:
Can we add a test for ErrDuplicateEntry?

Reply:
added a test case to check duplicate entry in cli tests in `TestCLITxGrantAuthorization`